src/js/init.js

```diff
@@ -16,6 +17,10 @@ module.exports = function(prefs) {
       isolate   = p.domain.isolate,
       lifetime  = p.lifetime;
 
+  // Select web storage method
+  storage_init.set(p.web_storage, p.session_length);
+  web_storage = storage_init.get();
```

Comment:
:question: 
I don't get the rationale behind this architecture, and it looks bug-prone and confusing to me.

1. In the PR description, you mentioned, that changing webstorage in runtime is currently not supported, and may result in corrupted data.
And there is nothing in this code that would prevent such corruption, or discourage it anyhow.
`sbjs.init()` may be called multiple times (and in WC Core we do so), that technically can happen with different prefs every time.
2. We set it to immediately get it at the line bellow and get it again inside the `migrations.go` another line bellow.

Do we even need that `storage_init` module? Can `sbjs.init` just have `web_storage` variable and pass it to `migrations.go`. 

I could imagine it would be useful, if it would not be `..._init` but a `storage` singleton, that would prevent from changing it in runtime.